### PR TITLE
Add Windows zip

### PR DESCRIPTION
**Pull request by ShiranAvidov**
I created a new Build and Archive for Windows, so each release will have Windows tar.gz and zip files.

#373

### CODE REVIEW

**Review by jpkrohling — APPROVED** [review]
I would be OK in giving this a try for v0.83.0. I would like to avoid merging this for the next release though (0.82.0), as it has the potential to break the already late current release.

**Comment by ShiranAvidov** [comment]
@jpkrohling  Is it possible to merge it to the next version (0.84.0)?

**Comment by jpkrohling** [comment]
I'll close and open this PR again, in an attempt to retrigger the CI: it has failed with a disk space issue, which I'm hoping is not tied to this PR.

**Comment by ShiranAvidov** [comment]
@jpkrohling no space left on device

**Comment by jpkrohling** [comment]
Can you check what are the disk requirements before and after this change? Are we just too close to the limit on the main branch, or is this change here causing too much usage?

File: .goreleaser.yaml [hunk 1]
```diff
@@ -1,12 +1,9 @@
-partial:
-  by: target
```
Comment (codeboten) [inline .goreleaser.yaml:2]: Why this change? This is here to avoid running into disk space issues

Comment (ShiranAvidov) [reply .goreleaser.yaml]: My bad. Was deleted by accident

**Comment by ShiranAvidov** [comment]
@codeboten @jpkrohling I added `partial` to the yaml.

File: .goreleaser.yaml (hunk 1 above)
Comment (codeboten) [reply .goreleaser.yaml]: np!

**Comment by ShiranAvidov** [comment]
@jpkrohling @codeboten  all checks have passed...

**Comment by atoulme** [comment]
would you please look at the conflicts?

**Comment by atoulme** [comment]
@ShiranAvidov please let us know if you would like to work to resolve conflicts on this PR, otherwise we will close it for inactivity in 30 days.

**Comment by andrzej-stencel** [comment]
Closing as inactive.
